HACKING.md

```diff
@@ -168,6 +168,11 @@ oc new-app --file=manifests/pipeline.yaml \
     --param=REPO_REF=my-feature-branch
 ```
 
+Optionally, the following parameters are supported:
+- `DISTRO`: build a different distribution, e.g `rhcos`
+- `NAMESPACE`: use a different namespace other than `fedora-coreos`
+- `VOLUME_SIZE`: define a different volume size for Jenkins, defaults to 50Gi
```

Comment:
should we mention `TAGS` here too?

Reply:
Fixed and re-pushed.

Reply:
i refreshed my browser. I still don't see `TAGS` mentioned here. Do you?

Reply:
Per another comment, TAGS are dropped in favor of hard-coded values.